### PR TITLE
[MRG] Uses approx to compare float in test_elbow

### PR DESCRIPTION
**Pull request by thomasjpfan**
This test is flaky on my local machine. This PR adds `pytest.approx` when comparing floats.

### CODE REVIEW

**Comment by lwgray** [comment]
@thomasjpfan thanks for this PR.  We look forward to reviewing it however we are on a short 2 week hiatus and will take a look at it when we return.  Thanks again for the PR

**Comment by rebeccabilbro** [comment]
Hey there @thomasjpfan &mdash; apologies for the slow response; we're still finishing up #776, which currently blocks moving forward with anything related to elbow. Once that is merged in, would you mind pulling from develop, rerunning the tests locally on your machine, and reporting back on what you observe?

**Comment by rebeccabilbro** [comment]
Ok @thomasjpfan, #776 is merged &mdash; can you go ahead and do a pull from `develop`, rerun the elbow tests, and let me know what you get?

**Comment by bbengfort** [comment]
Hi @thomasjpfan I think when you merged the `master` branch, it brought along some changes from our `v0.9.1` hotfix that shouldn't be in this PR. I know this is a lot of drama for a one line change (and a very good change at that) - but would you mind reverting to 79d0e62 and then merging with `develop` only? That should fix the CI tests and bring in the changes that @rebeccabilbro mentioned.

**Comment by thomasjpfan** [comment]
I just noticed that yellowbrick uses develop. 😅

**Comment by bbengfort** [comment]
@thomasjpfan yeh, we protect master as our release branch - the default branch on GitHub is develop. Let me know if you need assistance repairing your branch.

File: tests/requirements.txt [hunk 1]
```diff
@@ -18,4 +18,4 @@ nltk>=3.2
 # spacy>=2.0.18 
 pandas>=0.20
 umap-learn==0.3
-numba==0.42
\ No newline at end of file
+numba==0.42
```
Comment (bbengfort) [inline tests/requirements.txt]: This must be something to do with whitespace? I'm not 100% sure what is causing this line to change.

Comment (thomasjpfan) [reply tests/requirements.txt]: Newline was added in this PR. (Removed now)

Comment (bbengfort) [reply tests/requirements.txt]: Good eye! Sorry, I wasn't trying to nitpick about it - but thank you for making the update! We'll get this merged in as soon as the CI completes.

**Pull request merged**